src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,27 @@
+$nav-height: 60px;
+$nav-horizontal-p: 50px;
+
+$card-width: 200px;
+$cards-row: 4;
+$cards-gap: 48px;
+$cards-vertical-margin: 50px;
+
+$stars-size: 16px;
+
+$buy-height: 40px;
+
+$xs-text: 10px;
+$s-text: 12px;
+$m-text: 14px;
+$l-text: 16px;
+
+$weight-thin: 400;
+$weight-normal: 500;
+$weight-bold: 700;
```

Comment:
```suggestion

```

redundant, it is default variables

https://developer.mozilla.org/ru/docs/Web/CSS/font-weight